Makefile

```diff
@@ -604,6 +612,33 @@ $(SBOM): $(ROOTFS_TAR) | $(INSTALLER)
 	rm -rf $(TMP_ROOTDIR)
 	$(QUIET): $@: Succeeded
 
+$(GOSOURCES):
+	$(QUIET): $@: Begin
+	$(shell GOBIN=$(BUILDTOOLS_BIN)/yash GO111MODULE=on CGO_ENABLED=0 go install $(GOSOURCES_SOURCE)@$(GOSOURCES_VERSION))
```

Comment:
I think you meant for this to be `GOBIN=$(BUILDTOOLS_BIN)` and not `$(BUILDTOOLS_BIN)/yash`. 😁

Reply:
Yeah, you got that already

Reply:
this is fixed before the comment. Was just testing it with few combinations :)